doc/delivery.md

```diff
@@ -195,6 +195,14 @@ deliveryClient.items<Movie>()
   .getObservable()
   .subscribe(response => console.log(response));
 ```
+The following filters accept multiple query arguments within a list: `inFilter`, `allFilter`, `anyFilter`, `containsFilter`.
+```typescript
+deliveryClient.items<Movie>()
+  .type('movie')
+  .containsFilter('elements.title', ['Warrior', 'Monk'])
```

Comment:
@kentico-anthonym Technically, this method also contains 2 parameters :-) Its just that the second parameter expects an array to be passed.